servers/jax-rs-testextension/src/main/java/org/projectnessie/jaxrs/ext/NessieJaxRsExtension.java

```diff
@@ -149,7 +149,9 @@ public Object resolveParameter(
 
     if (parameterContext.isAnnotated(NessieUri.class)) {
       // Backward compatibility with older (external) tests
-      return getBaseUri(extensionContext);
+      // Inject v1 URIs in this case. Version-aware test should use
+      // `org.projectnessie.client.ext.NessieUri`.
```

Comment:
Should `org.projectnessie.client.ext.NessieUri` be renamed then to avoid ambiguouty?

Reply:
will do

Reply:
renamed